### PR TITLE
doc: connectivity: ble: initial basic profile instructions for Alif BLE stack

### PR DESCRIPTION
**Pull request by VeijoPesonen**
The instructions for Basic Profile Setup with Alif BLE stack. Instructions for audio use cases will be added in another PR,

I removed the Application Notes folder as unnecessary.

### CODE REVIEW

File: doc/links.txt [hunk 1]
```diff
@@ -0,0 +1,28 @@
+.. ### This page lists links used in Alif docs.
+
+.. _`Alif Toolkit Download`: https://alifsemi.com/download/APFW0002
+.. _`Alif Zephyr SDK Source`: https://github.com/AlifSemiDev/zephyr_alif
```
Comment (GaneshRamani-AlifSemi) [inline doc/links.txt:4]: need to change "AlifSemiDev" ?

Comment (VeijoPesonen) [reply doc/links.txt]: Thanks for spotting that one.

File: doc/README.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+|Folder         | Document Name     |
+|---------------|-------------------|
+| .             | SDK documentation |
+| user_guide    | User Guide        |
+| release_notes | Release Notes     |
+
+As of now you're supposed to build all the documents separately.
+
+Tested with Ubuntu 24.04. The goal is to build the documentation in html- and pdf-format.
```
Comment (HemanthKumar-AlifSemi) [inline doc/README.md:12]: @VeijoPesonen standard Ubuntu support for zas is 22.04.

Comment (VeijoPesonen) [reply doc/README.md]: Removed the sentence.

**Review by AnttiKauppila-AlifSemi — APPROVED** [review]

**Review by jheiskan81 — APPROVED** [review]

**Pull request merged**